### ISSUE TITLE
Change prompt to user-like prompt

### ISSUE THREAD

**Issue by pierrepo** [sandbox-bio/sandbox.bio#99]
The current prompt in the terminal is `root@localhost:~/tutorial#`

Would it be possible to change this prompt to a more user-like prompt?

- replace `#` by `$`
- change user from `root` to `robert`, `alice` or `bob`... 

Changing the login user might be the ultimate solution. It will also ensure the starting directory is `/home/robert/tutorial` instead of `/root/tutorial` that is not realistic.

**Comment by pierrepo** [comment]
Also changing `localhost` to `sandbox` could be nice ;-)

**Comment by robertaboukhalil** [comment]
Good idea, I wonder if it would be too slow to create creating custom users with `useradd` at runtime.
Changing `localhost` to `sandbox` is definitely doable :)

For the image issue mentioned in https://github.com/IFB-ElixirFr/sandboxbioscenarios/issues/21, I would recommend updating `<img>` to `<Image>`, I just pushed this change: https://github.com/sandbox-bio/sandbox.bio/commit/3de2d966cc05c7cf816f9949254f5e66c4b939dd. This makes the image larger and you can click on it to zoom it, as you can see at https://sandbox.bio/tutorials/ifb-linux-basics-2/1.

**Comment by RandyHarr** [comment]
To change the prompt, you can always just type: 
PS1="$ "    # or use whatever your favorite prompt is.
Not sure about the login user setup and whether a file is retained between sessions to store a change more permanently.

**Comment by robertaboukhalil** [comment]
The new sandbox.bio I deployed recently fixes this: it now shows `$` instead of `#` and uses user `sandbox` instead of `root`, thanks for the suggestion!
